edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Docker/DockerConfig.cs

```diff
@@ -20,21 +20,15 @@ public class DockerConfig : IEquatable<DockerConfig>
         readonly CreateContainerParameters createOptions;
 
         public DockerConfig(string image)
-            : this(image, string.Empty)
+            : this(image, string.Empty, null)
         {
         }
 
-        public DockerConfig(string image, string createOptions)
-            : this(ValidateAndGetImage(image), GetCreateOptions(createOptions))
+        public DockerConfig(string image, string createOptions, ContentTrust contentTrust)
+            : this(ValidateAndGetImage(image), GetCreateOptions(createOptions), Option.Maybe(contentTrust))
         {
         }
 
-        public DockerConfig(string image, CreateContainerParameters createOptions)
```

Comment:
Would there be any side effect by removing this function?

Reply:
Its a chained constructor. First we get the config as string and then we convert to CreateContainerParameters. So we need both